app/scripts/modules/core/src/pipeline/config/stages/deleteService/deleteServiceStage.ts

```diff
@@ -0,0 +1,18 @@
+import { module } from 'angular';
+
+import { Registry } from 'core/registry';
+import { ExecutionDetailsTasks } from '../core';
+import { DeleteServiceExecutionDetails } from './DeleteServiceExecutionDetails';
+
+export const DELETE_SERVICE_STAGE = 'spinnaker.core.pipeline.stage.deleteService';
+
+module(DELETE_SERVICE_STAGE, []).config(() => {
```

Comment:
Does this stage type show up if you don't have Cloudfoundry enabled? (There's a way to make it only show up if Cloudfoundry's enabled, but I don't remember the flags in the stage config).

Reply:
It does not show up if you don't enable Cloud Foundry.

The property I think you are referring to is `useBaseProvider: true`, which is configured.